### DISCUSSION TITLE
AddParameter does not create the column in my tracking table

### DISCUSSION THREAD

**Discussion by arjunbharara** [Mimetis/Dotmim.Sync#1278] [Q&A]
if (setup.Tables[tableForFilter.TableName] != null)
{
    var filter = new SetupFilter(tableForFilter.TableName);
    var columnName = clusterColumnSyncDbo.ColumnName.Trim();
    filter.AddParameter(columnName, tableForFilter.TableName);
    //clusterColumnSyncDbo.Filter.Trim();
    filter.AddCustomWhere("{{{IsSystemUser}}}= 0");
    setup.Filters.Add(filter);
}

the column does not get added in my tracking table .

**Comment by Mimetis** [comment]
tracking tables has only the primary key and the metadata columns

**Comment by arjunbharara** [comment]
I read this in the documentation and assumed it would behave the same. I believe the Microsoft Sync Framework also adds a column to the tracking table for the column used in the filter.
![docs](https://github.com/user-attachments/assets/7100fcb5-70dc-45b5-a60c-a8f3cbe95cbf)

Then how it handles the Filters?

**Reply by Mimetis** [reply]
There is no mention that the filter is added in the tracking tables in the documentation
Filters are only used in the stored procedures (especially the `GetChanges` stored procedures)

**Comment by arjunbharara** [comment]
but there you have mentioned that column will be automatically in the tracking table.

**Reply by Mimetis** [reply]
I don't think so
